app/Models/Server.php

```diff
@@ -125,7 +127,7 @@
  * @method static \Illuminate\Database\Eloquent\Builder|Server wherePorts($value)
  * @method static \Illuminate\Database\Eloquent\Builder|Server whereUuidShort($value)
  */
-class Server extends Model implements Validatable
+class Server extends Model implements HasAvatar, Validatable
```

Comment:
Doesn't need to implement `HasAvatar`. Filament will check the model for a `avatar_url` field and [use it automatically](https://filamentphp.com/docs/3.x/panels/users#:~:text=however%2C%20if%20your%20user%20model%20has%20an%20avatar_url%20attribute%2C%20that%20will%20be%20used%20instead.).